### PR TITLE
Create page fix

### PR DESCRIPTION
**Pull request by joac1144**

### CODE REVIEW

File: src/ProjectIT/Client/ProjectIT.Client.csproj
Comment (joac1144) [inline src/ProjectIT/Client/ProjectIT.Client.csproj:1]: Was automatically added by VS for Mac. Does not seem to break anything

File: src/ProjectIT/Server/ProjectIT.Server.csproj
Comment (joac1144) [inline src/ProjectIT/Server/ProjectIT.Server.csproj:1]: Was automatically added by VS for Mac. Does not seem to break anything

File: src/ProjectIT/Tests/Repository.Tests/TopicsRepositoryTests.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        );
+
+        context.SaveChanges();
+
+        _context = context;
+        _topicsRepository = new TopicsRepository(context);
+    }
+
+    [Fact]
+    public async void ReadAllAsync_ReturnsAllProjects()
```
Comment (Alaa0129) [inline src/ProjectIT/Tests/Repository.Tests/TopicsRepositoryTests.cs]: Fix method name to match test

File: src/ProjectIT/Tests/Controller.Tests/TopicsControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+using ProjectIT.Server.Controllers;
+using ProjectIT.Server.Repositories.Interfaces;
+using ProjectIT.Shared.Models;
+
+namespace Controller.Tests;
+
+public class TopicsControllerTests
+{
+    [Fact]
+    public async void GetAll_ReturnsProjects()
```
Comment (Alaa0129) [inline src/ProjectIT/Tests/Controller.Tests/TopicsControllerTests.cs]: Fix method name to match test

**Review by Alaa0129 — CHANGES_REQUESTED** [review]

File: src/ProjectIT/Tests/Repository.Tests/TopicsRepositoryTests.cs (hunk 1 above)
Comment (joac1144) [reply src/ProjectIT/Tests/Repository.Tests/TopicsRepositoryTests.cs]: Fixed

File: src/ProjectIT/Tests/Controller.Tests/TopicsControllerTests.cs (hunk 2 above)
Comment (joac1144) [reply src/ProjectIT/Tests/Controller.Tests/TopicsControllerTests.cs]: Fixed

**Review by Alaa0129 — APPROVED** [review]

**Review by Alaa0129 — APPROVED** [review]

**Pull request merged**